docker/oasis-net-runner/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM golang:1.19-buster AS oasis-core
```

Comment:
I think buster doesn't have recent enough glibc to work with 23.x oasis-core (GLIBC>=2.32)

Reply:
I guess that also fixture and genesis.json used in tests will require updates (no sure how these were generated, by hand?)

Reply:
buster: I was almost done debugging this when I saw your comment, and sure enough, the final run confirmed it :). Thank you.
As predicted, I then also had to update the fixture in a tiny way.